lmp/jobserv.yml

```diff
@@ -122,9 +122,9 @@ triggers:
       GIT_URL: |
         https://github.com/foundriesio/lmp-manifest.git
       GIT_POLL_REFS: |
-        refs/heads/main-next
-      OTA_LITE_TAG: 'main-next:main'
-      AKLITE_TAG: promoted-next
+        refs/heads/main
```

Comment:
At the moment we can only do releases based on main, I believe we decided to first release v95 out of main before switching main to the actual oe-core main, didn't we? @angolini ^

Reply:
Ok given that we can only do releases based on main , maybe is better to define two following path/branches for before and after v95:

**Before v95:**
build-scarthgap
`refs/heads/main`
build-main
`refs/heads/main-next`

I need to change the current PR to follow this.

**After v95:**
build-scarthgap
`refs/heads/scarthgap`
build-main
`refs/heads/main`

I create a new PR to be merged after v95 is released.

Reply:
I mean adding this on top of this PR and reverting it after v95 is released.
```diff
diff --git a/lmp/jobserv.yml b/lmp/jobserv.yml
index 31d0cc0..3164949 100644
--- a/lmp/jobserv.yml
+++ b/lmp/jobserv.yml
@@ -122,7 +122,7 @@ triggers:
       GIT_URL: |
         https://github.com/foundriesio/lmp-manifest.git
       GIT_POLL_REFS: |
-        refs/heads/scarthgap
+        refs/heads/main
       OTA_LITE_TAG: 'scarthgap'
       AKLITE_TAG: promoted-scarthgap
     runs:
@@ -227,7 +227,7 @@ triggers:
       GIT_URL: |
         https://github.com/foundriesio/lmp-manifest.git
       GIT_POLL_REFS: |
-        refs/heads/main
+        refs/heads/main-next
       OTA_LITE_TAG: 'main'
       AKLITE_TAG: promoted-main
     runs:

```

Reply:
updated in https://github.com/foundriesio/ci-scripts/pull/371/commits/f637aba38dd650c48e172dded1cec2cd4600064a

Reply:
With the launch of v95 I revert the https://github.com/foundriesio/ci-scripts/commit/f637aba38dd650c48e172dded1cec2cd4600064a

Reply:
I have droped the two coomits above https://github.com/foundriesio/ci-scripts/commit/f637aba38dd650c48e172dded1cec2cd4600064a and https://github.com/foundriesio/ci-scripts/commit/f637aba38dd650c48e172dded1cec2cd4600064a. We can forget all this path/branches for before and after v95.